web/packages/teleport/src/services/audit/types.ts

```diff
@@ -787,6 +791,54 @@ export type RawEvents = {
       }[];
     }
   >;
+  [eventCodes.DATABASE_SESSION_USER_CREATE]: RawEvent<
```

Comment:
we could create one base type to avoid repeating the types so much

```ts
type RawDatabaseSessionEvent<T extends EventCode, K = unknown> = Merge<
  RawEvent<
    T,
    {
      name: string;
      db_service: string;
      db_name: string;
      db_user: string;
      username: string;
    }
  >,
  K
>;
```

and then for other events where you get unique fields just for that event we can do:

```ts
[eventCodes.DATABASE_SESSION_USER_CREATE_FAILURE]: RawDatabaseSessionEvent<
    typeof eventCodes.DATABASE_SESSION_USER_CREATE_FAILURE,
    {
      error: string;
      message: string;
      roles: string[];
    }
  >;
```

Reply:
Nice, I'll do it. Thanks!